js/phobos.js

```diff
@@ -23,14 +23,28 @@ function addAnchors()
         // we link to the first children
         var a = items[i].querySelector('a');
         if(!a) continue;
+
+        // copy-to-clipboard
+        var el = document.createElement("i");
+        var symbol = a.name.replace(".", "");
```

Comment:
There is another anchor w/o the dot, on some other element. There are too many links in the internet to easily change this.